lib/host/wasi/inode-macos.cpp

```diff
@@ -1065,7 +1076,7 @@ WasiExpect<void> Poller::wait(CallbackType Callback) noexcept {
   const auto Count =
       ::kevent(Fd, nullptr, 0, KEvents.data(), KEvents.size(), nullptr);
   if (unlikely(Count < 0)) {
-    return WasiUnexpect(fromErrNo(errno));
+    return WasiUnexpect(fromEAIErrNo(errno));
```

Comment:
unnecessary change, use `fromErrNo`

Reply:
This change may be neccessary, because if we use `fromErrNo` function ,`netdb.h` defined the same errno values as `error.h`,which will lead to macos action fail.

 https://github.com/WasmEdge/WasmEdge/pull/573#issuecomment-994820753

> Looks like the `netdb.h` defined the same errno values as `error.h`. Please move the errno in `netdb.h` to another function. For example, `fromEAIErrNo()` to handle those errno from `netdb.h`.

Reply:
Sorry, I looked wrong line's number. You are correct.